source/_patterns/00-atoms/components/tweet.yaml

```diff
@@ -0,0 +1,31 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+properties:
+  url:
+    type: string
+    minLength: 1
+  accountId:
+    type: string
+    minLength: 1
+  accountLabel:
+    type: string
+    minLength: 1
+  text:
+    type: array
+    items:
+      $ref: ../../00-atoms/components/paragraph.yaml
+    minItems: 1
+  date:
+    $ref: ../../00-atoms/components/date.yaml
+  hideConversation:
```

Comment:
For consistency, prefer to use the `enum` approach for defining booleans, e.g. 

```YAML
hideConversation:
  type: boolean
  enum:
    - true
```
Also [see further example](https://github.com/elifesciences/pattern-library/blob/develop/source/_patterns/00-atoms/components/speech-bubble.yaml#L10 for example).

This has the advantage of if it's present, it's true, which plays nicely with mustache.